.eslintrc.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
in der Anwendung wird kein eslint benötigt

Reply:
Datei entfernt